### ISSUE TITLE
TikZ plugin is no-longer working in online example

### ISSUE THREAD

**Issue by warren-bank** [BearToCode/carta#88]
[example](https://beartocode.github.io/carta/examples#math-stack-exchange)

if I were to wager an off-hand guess,
I'd put my money on this being the result of how `browser` (vs. SSR) determination is done.

I'm still unclear on the downside to using: `import {browser} from '$app/environment'`.
Since every plugin receives an instance of `carta`,
and since `carta` has `svelte` as a dependency,
`carta` could expose the value of `browser`,
which would allow plugins to say: `if (carta.isBrowser) ...`

---

I just realized that you've been making changes to this plugin today.
Maybe it's a work-in-progress, in which case.. I spoke too soon (sorry).
As an fyi, I also just realized that the online example does initally render the "circuit diagram" (probably SSR)..
and then when the page rehydrates client-side.. it disappears.
so there's a flash of content.. and then it's gone.

**Comment by BearToCode** [comment]
Thanks for your report, that was my fault, I just fixed that. Didn't notice it when I made the change.

By the way, the issue with using `'$app/environment'` is that it's a SvelteKit only property. In fact, if you try to build the library with it, you get a warning saying that it's not recommended to use that, as you would need to have SvelteKit as a dependency.
I found that using `esm-env`, which was suggested as an alternative, works just as good!
